package/android/src/main/java/com/reactnativecommunity/slider/ReactSlider.java

```diff
@@ -66,6 +66,18 @@ public class ReactSlider extends AppCompatSeekBar {
 
   private List<String> mAccessibilityIncrements;
 
+  /** Real limit value based on min and max values. This comes from props */
+  private double mRealLowerLimit = Long.MIN_VALUE;
+
+  /** Limit based on progress from 0..100 */
+  private int mLowerLimit = 0;
+
+  /** Real limit value based on min and max values. This comes from props */
+  private double mRealUpperLimit = Long.MAX_VALUE;
+
+  /** Limit based on progress from 0..100 */
+  private int mUpperLimit = 100;
```

Comment:
Setting lowest and highest limit possible for when the limits are not set

Reply:
mUpperLimit range is from 0 to 100 (like a percentage). 100 being the rightmost value of the slider. It is set to 100 as default as this value has no effect on the slider.

Reply:
Actually, I'm testing because this might not be true... `mUpperLimit` could be more than 100 if there are more than 100 total steps.